crates/router/src/core/payment_methods/cards.rs

```diff
@@ -472,19 +472,25 @@ fn filter_pm_country_based(
 ) -> (Option<admin::AcceptedCountries>, Option<Vec<String>>, bool) {
     match (accepted_countries, req_country_list) {
         (None, None) => (None, None, true),
-        (None, Some(ref r)) => (None, Some(r.to_vec()), false),
+        (None, Some(ref r)) => (
```

Comment:
do we need `ref` seems you should be able to consume the value instead of `ref + to_owned`